src/pages/Lend/Filter/LendFilterMenu.vue

```diff
@@ -7,7 +7,8 @@
 				<kv-icon class="filter-toggle-chevron" name="large-chevron" />
 			</div>
 			<div id="filter-menu">
-				<filter-section-categories :result-count="10" />
+				<filter-section-categories :result-count="10" :open="true" />
```

Comment:
I don't think we were limiting the list on the comps...especially with the location stuff...

Reply:
`resultCount` actually runs the displayed total count of results here, it's a hardcoded placeholder for now until we implement it. Once we decide where to put that logic we can update/remove it as necessary.

Reply:
cool!